src/lib/db/api-token-store.ts

```diff
@@ -74,8 +86,14 @@ const toRow = (newToken: IApiTokenCreate) => ({
     alias: newToken.alias || null,
 });
 
-const toTokens = (rows: any[]): IApiToken[] => {
-    const tokens = rows.reduce(tokenRowReducer, {});
+const toTokens = (
+    rows: any[],
+    allowOrphanedWildcardTokens?: boolean,
```

Comment:
Can we make this non nullable? It's not public so we shouldn't break other things and I really, really want people to choose this behavior intentionally and have the type system scream at them if they don't realize a choice is being made for them